runtimes/pythonrt/testdata/trigger_webhook.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
I think this shouldn't be a Go program, but I won't block the PR on this. However, please add a package docstring (https://go.dev/doc/comment#package) that explains what this program does, and what uses it (make).

Reply:
if I understand right, the reason for this program is not to assume that user have curl..But since we already have python we may use standard python packages, e.g. urlib or http?

Reply:
this is a test utility @tebeka used

I don't see any problem with it, but wdyt ?